playbooks/robusta_playbooks/playbooks_utils.py

```diff
@@ -0,0 +1,175 @@
+        values_format=chart_values_format
+    )
+    chart_name = query_name if query_name else promql_query
+    svg_name = f"{chart_name}.svg"
+    return FileBlock(svg_name, chart.render())
+
+
+def create_resource_enrichment(
+    starts_at: datetime,
+    labels: Dict[Any, Any],
+    resource_type: ResourceChartResourceType,
+    item_type: ResourceChartItemType,
+    prometheus_url: Optional[str] = None,
+    graph_duration_minutes: Optional[int] = None
+) -> FileBlock:
+    ChartOptions = namedtuple('ChartOptions', ['query', 'values_format'])
+    combinations = {
+        (ResourceChartResourceType.CPU, ResourceChartItemType.Pod): ChartOptions(
+            query='sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace="$namespace", pod=~"$pod"})',
+            values_format=ChartValuesFormat.Plain
+        ),
+        (ResourceChartResourceType.CPU, ResourceChartItemType.Node): ChartOptions(
+            query='instance:node_cpu_utilisation:rate5m{job="node-exporter", instance=~"$node_internal_ip:[0-9]+", cluster=""} != 0',
+            values_format=ChartValuesFormat.Percentage
+        ),
+        (ResourceChartResourceType.Memory, ResourceChartItemType.Pod): ChartOptions(
+            query='sum(container_memory_working_set_bytes{job="kubelet", metrics_path="/metrics/cadvisor", pod=~"$pod", container!="", image!=""})',
+            values_format=ChartValuesFormat.Bytes
+        ),
+        (ResourceChartResourceType.Memory, ResourceChartItemType.Node): ChartOptions(
+            query='instance:node_memory_utilisation:ratio{job="node-exporter", instance=~"$node_internal_ip:[0-9]+", cluster=""} != 0',
+            values_format=ChartValuesFormat.Percentage
+        ),
+        (ResourceChartResourceType.Disk, ResourceChartItemType.Pod): None,
+        (ResourceChartResourceType.Disk, ResourceChartItemType.Node): ChartOptions(
+            query='sum(sort_desc(1 -(max without (mountpoint, fstype) (node_filesystem_avail_bytes{job="node-exporter", fstype!="", instance=~"$node_internal_ip:[0-9]+", cluster=""})/max without (mountpoint, fstype) (node_filesystem_size_bytes{job="node-exporter", fstype!="", instance=~"$node_internal_ip:[0-9]+", cluster=""})) != 0))',
+            values_format=ChartValuesFormat.Percentage
+        ),
+    }
+    combination = (resource_type, item_type)
+    chosen_combination = combinations[combination]
+    if not chosen_combination:
+        raise AttributeError(f'The following combination for resource chart is not supported: {combination}')
+    values_format_text = 'Utilization' if chosen_combination.values_format == ChartValuesFormat.Percentage else 'Usage'
+    graph_enrichment = create_graph_enrichment(
+        starts_at,
+        labels,
+        chosen_combination.query,
+        prometheus_url=prometheus_url,
+        graph_duration_minutes=graph_duration_minutes,
+        query_name=f'{resource_type.name} {values_format_text} for this {item_type.name.lower()}',
```

Comment:
I would phrase this as: Pod CPU Usage
In addition, I would leave an option to set a custom title for the chart, that comes from the action params.
For example, if I want to write:
OOM Killed pod memory usage
or something ..